CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+cmake_minimum_required(VERSION 3.10)
+
+project("tinyfsm"
+    LANGUAGES CXX
+    VERSION 1.0.1
```

Comment:
why 1.0.1 ? should be 0.3.2
Can we extract this from include/tinyfsm.hpp line 26 ?
If not, do you know of some common / portable place to define the version number?
I'd hate to maintain the version in multiple files.

the "library" (API compatibility) version would be 0.3